connector_dicom/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+ir_model_partner_dicom,partner_dicom_group_user,model_partner_dicom,base.group_partner_manager,1,1,1,1
+
```

Comment:
Add read access to Internal Users, or they might start getting access errors when opening partner forms